src/main/java/nextstep/courses/infrastructure/JdbcSessionRepository.java

```diff
@@ -0,0 +1,72 @@
+package nextstep.courses.infrastructure;
+
+import nextstep.courses.domain.session.*;
+import nextstep.courses.type.SessionStatus;
+import org.springframework.jdbc.core.JdbcOperations;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Repository;
+
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Repository("sessionRepository")
+public class JdbcSessionRepository implements SessionRepository {
+
+    private JdbcOperations jdbcTemplate;
```

Comment:
repository 구현체에서 이미지를 같이 조회해서 매핑해서 주는 방법도 있습니다.

```suggestion
    private JdbcOperations jdbcTemplate;
    private ImageRepository imageRepository;
```

인터페이스를 의존한 뒤, find 시에 가져와서 mapping 하는 방법도 있어요 :)
다만, 이렇게하면 repository 간의 의존관계가 생기기 때문에 이부분도 고민이 필요해요 🤔
그리고 커플링이 높아지는 단점이 생기고, 훗날 요구사항이 추가되면서 유지보수하기 힘들어질 수 있습니다 🫠
충분한 고민해보시길 바랄게요!

Reply:
미션을 하는것에선 service에서 강의와 이미지를 각각 조회해서 가지도록 했습니다. 
의존성을 높여서 쿼리를 줄이냐, 의존성을 줄여서 쿼리를 늘리냐 어디까지 허용해야할지 역시 고민이네요 ㅎㅎ😂